GNUmakefile

```diff
@@ -0,0 +1,50 @@
+# Makefile for the supdup server and client.
+
+SHELL = /bin/sh
+PREFIX ?= /usr/local
+
+OS_NAME = $(shell uname)
+
+CC ?= cc
+CFLAGS += $(shell pkg-config --cflags ncurses 2> /dev/null) -g -Wall
+LDFLAGS += -g
+
+# AIX needs linking libcurses *after* libncurses
```

Comment:
Why does AIX need both curses _and _ ncurses?

Reply:
I wish I knew.  It could probably be made to work with AIX xcurses (in base) rather than ncurses, but ncurses is an available package.